assets/stories/skate-design/Typography.mdx

```diff
@@ -0,0 +1,100 @@
+import { Meta } from "@storybook/blocks"
+import "../../css/storybook_reboot.scss"
+
+<Meta title="Typography" />
+
+# Global Settings / Style
+
+Examples of typography styles found in Skate, based on defaults from
+Reboot with some modifications.
+
+<div class="sb-unstyled">
```

Comment:
suggestion: Can we use `<Unstyled/>` here instead? 
https://storybook.js.org/docs/api/doc-block-unstyled

Reply:
Same for the rest of the document which uses this class

Reply:
Ah I missed that when skimming through the available doc blocks, I'll give it a try!